### DISCUSSION TITLE
시도했으나 성공하지 못한 문제를 가져올 수 있나요?

### DISCUSSION THREAD

**Discussion by tjrkdgnl** [solvedac/unofficial-documentation#36] [질답] [answered]
1. 시도했으나 성공하지 못한 문제를 API를 통해 가져올 수 있을까요?

2. 1.이 안된다면 시도했던 문제를 API를 통해 가져올 수 있는지 궁금합니다.

**Comment by RanolP** [comment, marked as answer]
`GET /verify_credentials` 에 관련 정보가 있는 듯 합니다.

추후 문서에 반영하겠습니다.

**Comment by RanolP** [comment]
30f1efa 에서 보강했습니다
